### ISSUE TITLE
Context not passed to auth mutations

### ISSUE THREAD

**Issue by joshourisman** [strawberry-graphql/strawberry-graphql-django#102]
I created a simple test in an existing Django project to simply try and authenticate in GraphQL with Strawberry. Unfortunately, my test always fails, because the `auth.login()` mutation receives `None` as the request context, resulting in `info.context.request` failing.

By overriding the `auth.login` mutation with my own, identical code, I was able to confirm that it is in fact the case that `info.context is None`.

Everything appears to be set up exactly as in the documentation, here's the relevant code:

```tests.py

from django.test import TestCase

from .schema import schema


class StrawberryTests(TestCase):
    def test_login(self):
        query = """
            mutation TestLoginMutation($username: String!, $password: String!) {
                login(username: $username, password: $password) {
                    username
                    email
                }
            }
        """

        result = schema.execute_sync(
            query,
            variable_values={"username": "foo", "password": "SuperSecurePassword1"},
        )

        self.assertIsNone(result.errors)
        print(result.data)
```

```types.py
import strawberry
from django.contrib.auth import get_user_model
from strawberry.django import auto


@strawberry.django.type(get_user_model())
class User:
    username: auto
    email: auto


@strawberry.django.input(get_user_model())
class UserInput:
    username: auto
    password: auto
```

```schema.py
import strawberry
from strawberry.django import auth

from .types import User, UserInput


@strawberry.type
class Query:
    me: User = auth.current_user()


@strawberry.type
class Mutation:
    login: User = auth.login()
    logout = auth.logout()


schema = strawberry.Schema(query=Query, mutation=Mutation)
```

**Comment by joshourisman** [comment]
Oh. Duh. I guess I needed to type it out to see what was going on. Obviously there is no request context, because there is no request, because I'm executing the mutation against the schema directly…

**Comment by joshourisman** [comment]
Ok, I have confirmed that the issue is simply that there isn't a request to begin with, so of course there won't be a request context. By manually adding one, I have resolved that issue (while introducing another one, because this new request context doesn't have a Django session, but that can be fixed as well). It might be useful to provide some sort of Django testing client, as graphene does, or some other mechanism to easily make queries in tests that will look like they're coming from real HTTP requests. For reference, this code solved the issue of info.context being None:

```        result = schema.execute_sync(
            query,
            variable_values={"username": "foo", "password": "SuperSecurePassword1"},
            context_value=StrawberryDjangoContext(
                request=HttpRequest, response=HttpResponse
            ),
        )
```
